generator/sbpg/targets/rust.py

```diff
@@ -351,6 +376,20 @@ def __init__(self, msg, package_specs, field):
         if self.options.get("fields", False):
             self.bitfield = get_bitfield(self)
 
+# pattern to capture {{groups}} for enriched message display
+ENRICH_PAT = re.compile("{{([^}]+)}}[.]*", re.S)
```

Comment:
Definitely would like to see some tests for all this

Reply:
What is the `[.]*` at the end doing?  followed by 0 or more of any character?  This is meaningless right.

Reply:
Also the re.S option is a bit weird since these fields shouldn't have newlines in them